### ISSUE TITLE
[How to sign up] SimpleReport sign-up checklist should be a list 

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#6352]
## Area or Component

`How to sign up` page in the static site. The section `SimpleReport sign-up checklist`. 

## Quick Summary

The content describe in the checklist section represents a list but has been build as checkboxes elements. Not using the correct semantic element causes the list to not be identify as such.

### How to Replicate
Check the page and run the structure guided test. Verify that the checklist list is not highlighted as a list.

## Accessibility Violation

[Content appears like a list but is not marked up as such.](https://docs.deque.com/issue-help/1.0.0/en/semantic-list)

## Action Requested
Modify the list DOM to use a semantic element instead. We could use an ordered `<ol>` or unordered `<ul>` [list](https://designsystem.digital.gov/components/list/).

## Screenshots (optional)
![Screenshot 2023-08-17 at 10 25 10 AM](https://github.com/CDCgov/prime-simplereport/assets/103958711/a4b647b1-508f-4d56-aa52-9638e08f81c3)

**Comment by johanna-skylight** [comment]
Currently waiting on instructions to either use an ordered or unordered list. Tagging @jayna-SkylightDigital for follow up!.

**Comment by jayna-SkylightDigital** [comment]
@johanna-skylight Since the original list wasn't numbered, let's use an unordered list. Thanks!

**Comment by johanna-skylight** [comment]
@jayna-SkylightDigital I have a PR for this but I am not able to tag you. Let me know if the change looks good [Fix](https://github.com/CDCgov/prime-simplereport-site/pull/592)

**Comment by jayna-SkylightDigital** [comment]
@johanna-skylight (sorry for the delay) - the [fix](https://github.com/CDCgov/prime-simplereport-site/pull/592) looks perfect. Thank you!
